hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OutboundResponseHandler.java

```diff
@@ -87,15 +119,123 @@ public boolean send(Response response, Address target) {
 
         byte[] bytes = serializationService.toBytes(response);
         Packet packet = new Packet(bytes, -1)
-                .setPacketType(Packet.Type.OPERATION)
+                .setPacketType(OPERATION)
                 .raiseFlags(FLAG_OP_RESPONSE);
 
         if (response.isUrgent()) {
             packet.raiseFlags(FLAG_URGENT);
         }
 
+        return transmit(target, packet);
+    }
+
+    private boolean sendNormalResponse(Address target, long callId, int backupAcks, boolean urgent, Object value) {
+        checkTarget(target);
+
+        Packet packet = toNormalResponsePacket(callId, (byte) backupAcks, urgent, value);
+
+        return transmit(target, packet);
+    }
+
+    Packet toNormalResponsePacket(long callId, int backupAcks, boolean urgent, Object value) {
+        byte[] bytes;
```

Comment:
Can we still remove them at all? They are part of 3.8 and with rolling upgrade we still have to support them (at least one minor version). Otherwise this should go on the 4.0 list.